CHANGELOG.md

```diff
@@ -5,11 +5,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased](https://github.com/contentful/contentful.php/compare/1.2.0...HEAD)
 
+### Added
+* `UploadFile` class now manages files which aren't yet processed (for preview API) **[BREAKING]**.
+
 ### Fixed
 * Retrieving a list of entries that contained multiple loops creates too many objects. **[BREAKING]** ([#105](https://github.com/contentful/contentful.php/pull/105))
   The new behavior is, that any entry that appears multiple times in the graph of the response will be the same instance.
 * The `contentful` script used to warm up/clear the cache was not marked as a binary in `composer.json` and thus not published to `vendor/bin`.
 
+### Changed
+* Moved file management classes to a sub-namespace `Contentful\File`.
```

Comment:
Please mark this as a breaking change (see some older entries for examples). Also just call out the classes (there are only 2) instead of calling it file management.